proto/api/v1/auth_service.proto

```diff
@@ -20,21 +21,149 @@ message UserAttribute {
 
 message AuthStatus {
     oneof id {
-        string                user_id     = 1;
-        storage.ServiceIdentity       service_id  = 2;
+        string                  user_id         = 1;
+        storage.ServiceIdentity service_id      = 2;
     }
-    google.protobuf.Timestamp expires     = 3;
-    string                    refresh_url = 4;
+    google.protobuf.Timestamp   expires         = 3;
+    string                      refresh_url     = 4;
+    storage.AuthProvider        auth_provider   = 5;
+    storage.UserInfo            user_info       = 6;
```

Comment:
do not use storage protos

Reply:
This is an existing API proto, it cannot be simply renamed / refactored.

Additionally, this is only here due to formatting changes and not in scope of this PR. I'm not willing to make any changes on this as the PR is already big and scheduled to land in 4.3.